### ISSUE TITLE
Windows rejects installer package

### ISSUE THREAD

**Issue by TurkeyMan** [dlang/visuald#285]
Wow, this is a whole new level of Microsoft wrecking everything!

![image](https://github.com/user-attachments/assets/34944675-bc84-4ce4-a1a6-258312269d25)

**Comment by rainers** [comment]
Argh. Signing the binary might help, but unfortunately the certificate of the foundation expired. I guess you can still run the installer from within VS if you switch the check for updates to "Daily pre-releases" and let it download the file.

**Comment by TurkeyMan** [comment]
Yeah I resolved the issue locally, I was just letting you know. Do you know the cost to renew the certificate? Dlang foundation will probably sponsor it.

**Comment by p0nce** [comment]
Certificates were super annoying and now probably easier to get with Azure signing.
